docs/flashbots-auction/searchers/advanced/rpc-endpoint.mdx

```diff
@@ -119,6 +119,86 @@ example response:
 }
 ```
 
+### eth_sendPrivateTransaction
+
+`eth_sendPrivateTransaction` is used to send a single transaction to Flashbots. Flashbots will attempt to send the transaction to miners for the next 25 blocks. See [Private Transactions](/flashbots-auction/searchers/advanced/private-transaction.mdx) for more info.
+
+This method has the following JSON-RPC format:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "eth_sendPrivateTransaction",
+  "params": [{
+    tx,               // String, raw signed transaction
+    maxBlockNumber,   // Number, optional. Highest block number in which the transaction should be included.
+  }]
```

Comment:
Please see the payload description in https://github.com/flashbots/flashbots-docs/pull/266 and add the info about fast mode.